hack/mock-unittest-data/yarn.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Good call, it is more readable. I'll fix that in a follow-up PR if you don't mind

Reply:
Since this change is related to generating test data sets and hence not crucial to the core logic being added (from functionality POV) I'm totally fine if you submit a follow-up PR for this.
I would have used a different argument if the new file were crucial to the core functionality though :) .